<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ramses</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icon-large</normaloff>:/icons/icon-large</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="mainStack">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <layout class="QVBoxLayout" name="mainLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <layout class="QVBoxLayout" name="duqf_settingsLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="mainStatusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogIn"/>
   <addaction name="actionSettings"/>
   <addaction name="actionAdmin"/>
   <addaction name="actionProjectSettings"/>
   <addaction name="actionPipeline"/>
   <addaction name="actionAssets"/>
   <addaction name="actionShots"/>
   <addaction name="actionTimeline"/>
   <addaction name="actionSchedule"/>
   <addaction name="actionStatistics"/>
  </widget>
  <action name="actionLogIn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/login</normaloff>:/icons/login</iconset>
   </property>
   <property name="text">
    <string>Log in...</string>
   </property>
  </action>
  <action name="actionLogOut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/logout</normaloff>:/icons/logout</iconset>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="actionUserProfile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/user-settings</normaloff>:/icons/user-settings</iconset>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </action>
  <action name="actionAdmin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="duqf_resources.qrc">
     <normaloff>:/icons/settings</normaloff>:/icons/settings</iconset>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </action>
  <action name="actionProjectSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/project</normaloff>:/icons/project</iconset>
   </property>
   <property name="text">
    <string>Project</string>
   </property>
   <property name="toolTip">
    <string>Project Admin</string>
   </property>
  </action>
  <action name="actionUserFolder">
   <property name="icon">
    <iconset resource="duqf_resources.qrc">
     <normaloff>:/icons/folder</normaloff>:/icons/folder</iconset>
   </property>
   <property name="text">
    <string>User folder</string>
   </property>
  </action>
  <action name="actionPipeline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/steps-menu</normaloff>:/icons/steps-menu</iconset>
   </property>
   <property name="text">
    <string>Pipeline</string>
   </property>
  </action>
  <action name="actionAssets">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/assets-table</normaloff>:/icons/assets-table</iconset>
   </property>
   <property name="text">
    <string>Assets</string>
   </property>
  </action>
  <action name="actionShots">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/shots-table</normaloff>:/icons/shots-table</iconset>
   </property>
   <property name="text">
    <string>Shots</string>
   </property>
   <property name="toolTip">
    <string>Shots</string>
   </property>
  </action>
  <action name="actionSchedule">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/calendar</normaloff>:/icons/calendar</iconset>
   </property>
   <property name="text">
    <string>Schedule</string>
   </property>
   <property name="toolTip">
    <string>Schedule</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/stats</normaloff>:/icons/stats</iconset>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionTimeline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/timeline</normaloff>:/icons/timeline</iconset>
   </property>
   <property name="text">
    <string>Timeline</string>
   </property>
   <property name="toolTip">
    <string>All shots in a timeline view</string>
   </property>
  </action>
  <action name="actionSetOnline">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/server-settings</normaloff>:/icons/server-settings</iconset>
   </property>
   <property name="text">
    <string>Set online</string>
   </property>
   <property name="toolTip">
    <string>Activate sync</string>
   </property>
  </action>
  <action name="actionSetOffline">
   <property name="icon">
    <iconset resource="duqf_resources.qrc">
     <normaloff>:/icons/folder</normaloff>:/icons/folder</iconset>
   </property>
   <property name="text">
    <string>Set offline</string>
   </property>
   <property name="toolTip">
    <string>Deactivates sync to work offline.</string>
   </property>
  </action>
  <action name="actionDatabaseSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/storage-settings</normaloff>:/icons/storage-settings</iconset>
   </property>
   <property name="text">
    <string>Database settings</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="duqf_resources.qrc">
     <normaloff>:/icons/settings</normaloff>:/icons/settings</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Ramses Application settings</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="icon">
    <iconset resource="duqf_resources.qrc">
     <normaloff>:/icons/check-update</normaloff>:/icons/check-update</iconset>
   </property>
   <property name="text">
    <string>Quick sync</string>
   </property>
   <property name="toolTip">
    <string>Quickly syncs the most recent data with the server.</string>
   </property>
  </action>
  <action name="actionFullSync">
   <property name="icon">
    <iconset resource="duqf_resources.qrc">
     <normaloff>:/icons/check-update</normaloff>:/icons/check-update</iconset>
   </property>
   <property name="text">
    <string>Full sync</string>
   </property>
   <property name="toolTip">
    <string>Syncs all the data with the server.</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="duqf_resources.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
